.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 node_js:
-  - iojs
-  - "0.12"
+  - "5.1.0"
+
+cache:
+  directories:
+    - node_modules
```

Comment:
don't cache node_modules, it will cause more problems, than its supposed to solve